tests/misc.rs

```diff
@@ -602,3 +602,126 @@ fn test_verifier_fail() {
     vm.set_program(&prog).unwrap();
     assert_eq!(vm.execute_program().unwrap(), 0xBEE);
 }
+
+#[test]
+fn test_vm_bpf_to_bpf_call(){
+    let test_code = assemble(
+        "
+        mov64 r1, 0x10
+        mov64 r2, 0x1
+        callx 0x4
+        mov64 r1, 0x1
+        mov64 r2, r0
+        callx 0x4
+        exit
+        mov64 r0, r1
+        sub64 r0, r2
+        exit
+        mov64 r0, r2
+        add64 r0, r1
+        exit
+        ").unwrap();
+    let vm = rbpf::EbpfVmNoData::new(Some(&test_code)).unwrap();
+    let vm_res= vm.execute_program().unwrap();
+    assert_eq!(vm_res, 0x10);
+}
+
+#[test]
+fn test_vm_jit_bpf_to_bpf_call(){
```

Comment:
Thanks for moving these! Contrarily to the test file with the JIT tests inherited from uBPF, this file doesn't have a toplevel `#![cfg(all(not(windows), feature = "std"))]
`, which means we must manually exclude tests using the JIT from `windows` and `no_std`. This should fix the Windows build.


```suggestion
#[cfg(all(not(windows), feature = "std"))]
#[test]
fn test_vm_jit_bpf_to_bpf_call(){
```